IdleLoop/main.cpp

```diff
@@ -0,0 +1,31 @@
+#include "mbed.h"
+#include "rtos_idle.h"
+
+DigitalOut led(LED2);
+DigitalOut led1(LED1);
```

Comment:
Two LEDs?

Reply:
I'm gonna nit. Mind updating the names so that they're a bit more descriptive?

At least something like `led2` and `led1`.

Reply:
You are right! I will probably add some more comments to this. Working on it